public/config.js.example

```diff
@@ -22,9 +22,12 @@ const config = {
   ENABLE_PARCEL: false,
   ENABLE_POINTER: false,
 
-  // The zoom difference between skraafotos and the WMTS service used for the map
-  ZOOM_DIFFERENCE: 12, // use 15.5 to have the same zoom on the map as the skraafotos
-  DEFAULT_ZOOM: 4, // the default zoom level for skraafotos
+  // The zoom difference between skraafotos and the WMTS service used for the map.
```

Comment:
Should I just remove these 5 zoom settings from the config files and only have them in `src/modules/configuration.js`? I doubt we will want different options for different solutions, but I guess it's possible.

Reply:
Do it. We can always add more external configuration options later.